tsl/test/isolation/specs/CMakeLists.txt

```diff
@@ -44,7 +20,30 @@ list(
 
 if(CMAKE_BUILD_TYPE MATCHES Debug)
   list(APPEND TEST_TEMPLATES_MODULE ${TEST_TEMPLATES_MODULE_DEBUG})
-  list(APPEND TEST_FILES cagg_multi_dist_ht.spec)
+  list(
+    APPEND
+    TEST_FILES
+    cagg_drop_chunks_iso.spec
+    cagg_multi_dist_ht.spec
+    compression_chunk_race.spec
+    compression_merge_race.spec
+    decompression_chunk_and_parallel_query_wo_idx.spec
+    dist_ha_chunk_drop.spec
+    dist_ha_chunk_drop.spec
+    dist_restore_point.spec
+    remote_create_chunk.spec)
+  if(PG_VERSION VERSION_GREATER_EQUAL "14.0")
+    list(APPEND TEST_FILES freeze_chunk.spec compression_dml_iso.spec)
+  endif()
+  # These tests are using markers for the isolation tests (to avoid race
+  # conditions causing differing output), which were added after 12.7, 13.3, and
+  # 14.0.
+  if(PG_VERSION VERSION_GREATER "12.7"
```

Comment:
We removed support for PG12 recenlty

Reply:
Test is harmless, but it is trivial to remove. Might be good to still keep the comment though, in case anybody want to test a build with deprecated versions.

Reply:
Leaving it for now. It is trivial to remove, but will generate weird errors if somebody tries to use it for PG12 even though it is deprecated.